### ISSUE TITLE
Rails/DuplicateAssociation  Autocorrect parameter set to false but not available

### ISSUE THREAD

**Issue by jenerts-at-levelpath** [standardrb/standard-rails#23]
Running Rubocop triggers warning: 

```
Warning: Rails/DuplicateAssociation does not support Autocorrect parameter.

Supported parameters are:

  - Enabled
  - Severity
```

looking into[ base config ](https://github.com/standardrb/standard-rails/blob/2c1dbf5e5bee3db4fef43da6d9d72f7543985cd5/config/base.yml#L135-L137) and indeed autocorrect is set to `false`  but documentation says that there is no such [parameter](https://docs.rubocop.org/rubocop-rails/cops_rails.html#configurable-attributes-17) 

Is this expected or a minor mistake in recent release? 

My  version of rubocop and plugins
```
bundle exec rubocop -V

1.59.0 (using Parser 3.2.2.4, rubocop-ast 1.30.0, running on ruby 3.2.2) [aarch64-linux]
  - rubocop-graphql 1.4.0
  - rubocop-performance 1.20.1
  - rubocop-rails 2.23.1
  - rubocop-rspec 2.25.0
  ```

**Comment by narnd** [comment]
There is also a duplication which throws:
``Rails/StripHeredoc` is concealed by line 373`

https://github.com/standardrb/standard-rails/blob/2c1dbf5e5bee3db4fef43da6d9d72f7543985cd5/config/base.yml#L370-L374

**Comment by searls** [comment]
Fixed by #24
